syncscribble/Makefile

```diff
@@ -99,7 +100,7 @@ DEFS += _USE_MATH_DEFINES UNICODE NOMINMAX FONS_WPATH
 
 # only dependencies under this path will be tracked in .d files; note [\\] must be used for "\"
 # ensure that no paths containing spaces are included
-DEPENDBASE ?= c:[\\]temp[\\]styluslabs
+DEPENDBASE ?= \\wsl.localhost\Ubuntu\home\zaitam\dev\forks\Write
```

Comment:
Is this an accident?

Reply:
Yes. I thought I did a rollback prior to the commit while I changed computers (Seems I forgot).